src/render/shaders/tilemap_vertex.wgsl

```diff
@@ -1,44 +1,49 @@
-#import bevy_ecs_tilemap::common
+#import bevy_ecs_tilemap::common VertexInput, tilemap_data
```

Comment:
We need `tilemap_data` for the `atlas` feature.
```
#import bevy_ecs_tilemap::common tilemap_data
```

Reply:
I'm unclear what you are asking to be changed here?  Does this import not already include tilemap_data or are you saying it should not?

Reply:
My bad, I meant to place that comment in `tilemap_fragment.wgsl`.